src/evaluators/source-4-3.js

```diff
@@ -0,0 +1,808 @@
+function is_true(x) {
```

Comment:
Add a description of the language covered, like in the virtual machines.

Reply:
Description added in a15a077